### ISSUE TITLE
Improve Chrome/Edge extension distribution for side loading

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-js#666]
Following the recent comment of @Jaifroid https://github.com/kiwix/kiwix-js/issues/624#issuecomment-711290059 I have a few (easy) questions and decisions to be made:
* What is the best sideload package to be provided for Chrome and how to get it.
* What is the best sideload package to be provided for Edge and how to get it.

If we should not provide `crx` files anymore for none of them, I need to know where to get the ZIP file. So far I have downloaded the CRX extension with a dedicated chromium extension... but this extension seems to not work to get a ZIP file (and it seems we need the unsigned version anyway... which is surprising to me).

Currently, you can look at what we provide (for the 3 browsers) https://download.kiwix.org/release/browsers/. Basically the edge extension is a symbolic link on the chrome one (with an other name).

**Comment by Jaifroid** [comment]
@kelson42 I've now tested these, and despite what it says in the article I linked, it is possible to install the .crx file we provide by going to `edge://extensions`, setting Developer mode, and then dragging the .crx file onto that page. It asks if the user wishes to install and then installs it.

Now, I don't have Chrome installed on my system (no need for two Chromium browsers), but since this is the Chrome store's .crx I can't imagine there's any problem installing it on Chrome.

In sum, I think everything is fine, and you don't need to change anything in those download links! (And apologies for misleading you with that web page -- which also misled me.)
